app/assets/javascripts/ama_layout/forms/placeholder.coffee

```diff
@@ -0,0 +1,30 @@
+class AMALayout.Placeholder
+  constructor: (args = {}) ->
+    args = $.extend({}, @defaultVariables(), args)
+    @placeholderTextColor = args.placeholderTextColor
+    @inputTextColor = args.inputTextColor
+    @defaultTextList = args.defaultTextList
+    @setup()
+
+  setup: ->
+    for defaultText in @defaultTextList
+      @changeDefaultPlaceholderColor(defaultText)
+      @updatePlaceholderColor(defaultText)
+
+  changeDefaultPlaceholderColor: (defaultText) ->
+    $("option:selected:contains('#{defaultText}')").parent('select').css('color', @placeholderTextColor)
```

Comment:
If you change this function accept a `select` element, you could move the color logic and property setting into here, then call it from the setup function and in the the change event. That'll reduce a bit of the CSS selector code, and makes for a reuasable function.